src/lib/paginator/paginator.html

```diff
@@ -0,0 +1,39 @@
+<div class="mat-paginator-page-length-select">
+  <div class="mat-paginator-items-per-page-label">
+    {{_intl.itemsPerPageLabel}}
+  </div>
+
+  <md-select *ngIf="_displayedPageLengthOptions.length > 1"
+             [ngModel]="pageLength"
+             [aria-label]="_intl.itemsPerPageLabel"
+             (change)="_changePageLength($event.value)">
+    <md-option *ngFor="let pageLengthOption of _displayedPageLengthOptions" [value]="pageLengthOption">
+      {{pageLengthOption}}
+    </md-option>
+  </md-select>
+
+  <div *ngIf="_displayedPageLengthOptions.length <= 1">{{pageLength}}</div>
+</div>
+
+<div class="mat-paginator-range-label">
+  {{_intl.getRangeLabel(currentPageIndex, pageLength, listLength)}}
+</div>
+
+<button md-icon-button
+        class="mat-paginator-navigation-button mat-paginator-navigation-previous"
+        (click)="navigateToPreviousPage()"
+        [attr.aria-label]="_intl.previousPageLabel"
+        [mdTooltip]="_intl.previousPageLabel"
+        [mdTooltipPosition]="'above'"
+        [disabled]="!canNavigateToPreviousPage(-1)">
+  <div class="mat-paginator-increment"></div>
+</button>
+<button md-icon-button
```

Comment:
We added the following to the `@Component`:

```js
  providers: [
    {provide: MATERIAL_COMPATIBILITY_MODE, useValue: false}
  ],
```

Which should make sure that the paginator's template will *not* be in compatibility mode.